.devcontainer/.pre-commit-config.yaml

```
@@ -0,0 +1,87 @@
+      - id: check-github-actions
+      - id: check-github-workflows
+        args: ["--verbose"]
+  - repo: https://github.com/python-poetry/poetry
+    rev: 1.8.0
+    hooks:
+      - id: poetry-check
+      - id: poetry-lock
+      - id: poetry-export
+        args: [
+            # TODO: This should be updating corresponding ./{partcad,docs,partcad-cli}/requirements.txt
+            "--format=requirements.txt",
```

yamllint (1.35.1)

[error] wrong indentation: expected 10 but found 12

(indentation)